base/src/main/scala/data/Const.scala

```diff
@@ -0,0 +1,8 @@
+package scalaz
+package data
+
+final case class  Const[A, B](getConst: A){
+  def retag[C]: Const[A, C] = this.asInstanceOf[Const[A, C]]
```

Comment:
Oh nice! Thanks for the reference, I didn't know about this one but I suppose that's the one @S11001001 mentioned to me during lambda conf.